src/gam.py

```diff
@@ -13126,6 +13126,7 @@ def _update_option(self, option, selected=None, restriction=None):
       if selected is None:
         # Toggle the option on/off
         option.is_selected = not option.is_selected
+        option._restriction = None
```

Comment:
Please use the accessor here, instead of setting a private member: option.unrestrict()

Reply:
Also, it looks like this restriction removal should be happening automatically in the is_selected property setter, when is_selected is set to False, so I'll have to look into why it doesn't seem to be working correctly.